docs/content.zh/release-notes/flink-1.19.md

```diff
@@ -0,0 +1,362 @@
+
+According to [FLIP-372](https://cwiki.apache.org/confluence/display/FLINK/FLIP-372%3A+Enhance+and+synchronize+Sink+API+to+match+the+Source+API) the SinkV2 API has been changed.
+The following interfaces are deprecated: `TwoPhaseCommittingSink`, `StatefulSink`, `WithPreWriteTopology`, `WithPreCommitTopology`, `WithPostCommitTopology`.
+The following new interfaces have been introduced: `CommitterInitContext`, `CommittingSinkWriter`, `WriterInitContext`, `StatefulSinkWriter`.
+The following interface method's parameter has been changed: `Sink#createWriter`
+The original interfaces will remain available during the 1.19 release line, but they will be removed
+in consecutive releases. For the changes required when migrating, please consult the Migration Plan detailed in the FLIP.
+
+
+#### Deprecate configuration getters/setters that return/set complex Java objects
+
+##### [FLINK-33581](https://issues.apache.org/jira/browse/FLINK-33581)
+
+The non-ConfigOption objects in the `StreamExecutionEnvironment`, `CheckpointConfig`, and 
+`ExecutionConfig` and their corresponding getter/setter interfaces are now be deprecated in [FLINK-33581](https://issues.apache.org/jira/browse/FLINK-33581).
+And these objects and methods are planned to be removed in Flink 2.0. The deprecated interfaces
+include the getter and setter methods of `RestartStrategy`, `CheckpointStorage`, and `StateBackend`.
+More details can be found at [FLIP-381](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=278464992).
+
+
+### Table SQL / API
+
+#### Support Setting Parallelism for Table/SQL Sources
+
+##### [FLINK-33261](https://issues.apache.org/jira/browse/FLINK-33261)
+
+Scan table sources can now be set a custom parallelism for performance tuning via the `scan.parallelism`
+option. Currently, only the DataGen connector has been adapted to support that, Kafka connector is
+on the way. Please check [scan-table-source](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/sourcessinks/#scan-table-source) on how to adapt your custom connectors to it.
+
+#### Adding a separate configuration for specifying Java Options of the SQL Gateway
+
+##### [FLINK-33203](https://issues.apache.org/jira/browse/FLINK-33203)
+
+Flink introduces `env.java.opts.sql-gateway` for specifying the Java options for the SQL Gateway,
+which allows you to fine-tune the memory settings, garbage collection behavior, and other relevant Java parameters.
+
+#### Support Configuring Different State TTLs using SQL Hint
+
+##### [FLINK-33397](https://issues.apache.org/jira/browse/FLINK-33397)
+
+This is a new feature in Apache Flink 1.19 that enhances the flexibility and user experience when
+managing SQL state time-to-live (TTL) settings. Users can now specify custom TTL values for regular
+joins and group aggregations directly within their queries by [utilizing the STATE_TTL hint](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/sql/queries/hints/#state-ttl-hints).
```

Comment:
Same here, I think using the link of `https://nightlies.apache.org/flink/flink-docs-release-1.19/docs/dev/table/sql/queries/hints/#state-ttl-hints` could be better.